src/main/kotlin/ee/carlrobert/codegpt/codecompletions/CodeGPTInlineCompletionProvider.kt

```diff
@@ -43,11 +43,14 @@ class CodeGPTInlineCompletionProvider : InlineCompletionProvider {
             currentCall.set(
                 CompletionRequestService.getInstance().getCodeCompletionAsync(
                     infillRequest,
-                    CodeCompletionEventListener(infillRequest) {
-                        request.editor.putUserData(CodeGPTKeys.PREVIOUS_INLAY_TEXT, it)
+                    CodeCompletionEventListener(infillRequest) { suggestion, needCancel ->
```

Comment:
Wouldn't `message, cancel` be more self-explaining than `suggestion, needCancel`?
```suggestion
                    CodeCompletionEventListener(infillRequest) { message, cancel ->
```

Reply:
Yeah! fixed.